hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/DirectoryDeletingService.java

```diff
@@ -154,11 +154,11 @@ public int getPriority() {
     @Override
     public BackgroundTaskResult call() {
       if (shouldRun()) {
+        isRunningOnAOS.set(true);
+        final long run = getRunCount().incrementAndGet();
         if (LOG.isDebugEnabled()) {
-          LOG.debug("Running DirectoryDeletingService");
+          LOG.debug("Running DirectoryDeletingService {}", run);
```

Comment:
We already log the run count in optimizeDirDeletesAndSubmitRequest() present inAbstractKeyDeletingService.java, do we need it here as well?

Reply:
In order to keep the log consistent across the various deleting services, I have added this. We can easily identify each run of every deleting service instead of having to search in the logs.